src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Cookies.cs

```diff
@@ -193,26 +178,15 @@ public async Task GetAsync_AddMultipleCookieHeaders_CookiesSent()
                 });
         }
 
-        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Netfx handler does not support custom cookie header")]
+        [ConditionalFact]
```

Comment:
Why [ConditionalFact] here?

(Actually I'm not sure why you can use [ConditionalFact] without any parameters, what does this even do?)

Reply:
This allows you to use "throw new SkipTestException" to dynamically skip instead of getting "Pass". This is somewhat recent update to MS XUnit extensions.